bpf/lib/lb.h

```diff
@@ -510,8 +510,8 @@ lb6_fill_key(struct lb6_key *key, struct ipv6_ct_tuple *tuple)
  *   - Negative error code
  */
 static __always_inline int
-lb6_extract_tuple(struct __ctx_buff *ctx, struct ipv6hdr *ip6, int *l4_off,
-		  struct ipv6_ct_tuple *tuple)
```

Comment:
The function comment above needs an update. It would also be good to mention there that `l4_off` is only for unit tests. Some people have been know to remove those seemingly-unused `l4_off` arguments in the past :innocent:

Reply:
FWIW, I expect that we'll end up using the l3_off parameter in other areas too (eg. see https://github.com/cilium/cilium/pull/18414#discussion_r1106777228). So think we won't need that comment - the unit tests will complain loud enough if they fail to compile :).

Reply:
Fair enough.